### ISSUE TITLE
Paradigms - Masternarde? greek-inflexion?

### ISSUE THREAD

**Issue by jonathanrobie** [deep-reader/DeepReader#33]
I would like to be able to quickly access paradigms for words.  Masternarde's paradigms are a useful resource for this.  You probably have better based on greek-inflexion.

**Comment by jtauber** [comment]
Yeah, I've started thinking about an API that does paradigms and also pivots around each property so if you have a PAI.3S verb it shows you:

- IAI.3S
- FAI.3S
- AAI.3S
- XAI.3S
- PMI.3S
- PMS.3S
- PMO.3S
- PMD.3S
- PAN
- PAP.NSM
- PAP.GSM
- PAI.1S
- PAI.2S
- PAI.3P
